packages/@aws-cdk/aws-ecs/lib/fargate/fargate-task-definition.ts

```diff
@@ -59,6 +60,15 @@ export interface FargateTaskDefinitionProps extends CommonTaskDefinitionProps {
    * @default 20
    */
   readonly ephemeralStorageGiB?: number;
+
+  /**
+   * The operating system that your task definitions are running on.
+   *
+   * NOTE: A platform family is specified only for tasks using the Fargate launch type.
```

Comment:
Let's just use the exact same wording as in task-definition.ts:
```suggestion
   * A runtimePlatform is supported only for tasks using the Fargate launch type.
```